core/workspace_svg.ts

```diff
@@ -1050,9 +1050,6 @@ export class WorkspaceSvg extends Workspace implements IASTNodeLocationSvg {
     if (this.toolbox_) {
       this.toolbox_.position();
     }
-    if (this.flyout) {
```

Comment:
You still need to handle the case where there's no toolbox but there is a flyout. So you want an else if, rather than just removing this.

Reply:
Ah good call, done1